docs/conf.py

```diff
@@ -10,100 +10,211 @@
-    'cartopy': ('https://scitools.org.uk/cartopy/docs/latest/', None)
+    'pint': ('https://pint.readthedocs.io/en/stable/', None),
+    'cftime': ('https://unidata.github.io/cftime/', None),
+}
+# allows us to easily link PRs and issues in the change log
+extlinks = {
+    "issue": ("https://github.com/NCAR/geocat-viz/issues/%s", "GH"),
+    "pr": ("https://github.com/NCAR/geocat-viz/pull/%s", "PR"),
 }
 
+# napoleon settings
 napoleon_use_admonition_for_examples = True
 napoleon_include_special_with_doc = True
+napoleon_google_docstring = False
+napoleon_numpy_docstring = True
+napoleon_use_param = False
+napoleon_use_rtype = False
+napoleon_preprocess_types = True
+napoleon_type_aliases = {
+    # general terms
+    "sequence": ":term:`sequence`",
+    "iterable": ":term:`iterable`",
+    "callable": ":py:func:`callable`",
+    "array": ":term:`array`",
+    "dict_like": ":term:`dict-like <mapping>`",
+    "dict-like": ":term:`dict-like <mapping>`",
+    "path-like": ":term:`path-like <path-like object>`",
+    "mapping": ":term:`mapping`",
+    "file-like": ":term:`file-like <file-like object>`",
+    "int": ":class:`int`",
+    "float": ":class:`float`",
+    "str": "str",
+    "bool": ":class:`bool`",
+    "tuple": ":class:`tuple`",
+    "list": ":class:`list`",
+    "dict": ":class:`dict`",
+    "set": ":class:`set`",
+    # numpy terms
+    "array_like": ":term:`array_like`",
+    "array-like": ":term:`array-like <array_like>`",
+    "ndarray": "numpy.ndarray",
+    "numpy.nan": "~numpy.nan",
+    "np.nan": "~numpy.nan",
+}
 
 autosummary_generate = True
 
-autodoc_default_options = {'exclude-members': '__init__'}
-
-autodoc_mock_imports = ['pint', 'xarray', 'metpy']
-
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
 
 # The suffix(es) of source filenames.
-source_suffix = '.rst'
+# You can specify multiple suffix as a list of string:
+# source_suffix = ['.rst', '.md']
+source_suffix = {
+    '.rst': 'restructuredtext',
+    '.ipynb': 'myst-nb',
+    '.myst': 'myst-nb',
+}
+
+# The encoding of source files.
+#source_encoding = 'utf-8-sig'
 
 # The master toctree document.
 master_doc = 'index'
 
+# General information about the project.
+project = u'GeoCAT-viz'
 
-# The version info for the project being documented
-def read_version():
-    for line in open('../meta.yaml').readlines():
-        index = line.find('set version')
-        if index > -1:
-            return line[index + 15:].replace('\" %}', '').strip()
-
+import datetime
 
-# The short X.Y version.
-version = read_version()
+current_year = datetime.datetime.now().year
+copyright = u'{}, University Corporation for Atmospheric Research'.format(
+    current_year)
+author = u'GeoCAT'
 
+# The version info for the project being documented
+version = gv.__version__.split("+")[0]
 # The full version, including alpha/beta/rc tags.
-release = read_version()
+release = gv.__version__
```

Comment:
This actually won't work in geocat-viz. In comp, this was actually a fairly recent change when we did the packaging and namespace rearrangement. Geocat-viz won't be able to do `gv.__version__`.

Reply:
Oh shoot! I didn't realize you'd also done the packaging structure changes. Could you please add that to the PR description?